package.json

```
@@ -32,6 +32,12 @@
       "title": "Departures",
       "description": "Search for public transport departures from any stop place in Norway",
       "mode": "view"
+    },
+    {
+      "name": "plan",
+      "title": "Plan Trip",
```

I think "Trip Search" is a bit more specific here.

```suggestion
      "title": "Trip Search",
```

---

> **Command title**
> Usually it's `<verb> <noun>` structure or just `<noun>`
> The best approach is to see how other commands are named in Raycast to get inspiration
> ✅ `Search Recent Projects`, `Translate`, `Open Issues`, `Create Task`
> ❌ `Recent Projects Search`, `Translation`, `New Task`
 
-- from [raycast's guide](https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming)

Maybe `Search Trip`?

---

Ah, thanks for looking up the guidelines! I'm more used to saying "Trip Search", but I guess it's better to be in line with other commands 👍 We should probably use plural tho, in line with other commands like "Search Notes" and "Search Snippets".

```suggestion
      "title": "Search Trips",
```